packages/web/server/api/routers/bridge-transfer.ts

```diff
@@ -20,6 +20,7 @@ import { CacheEntry } from "cachified";
 import { LRUCache } from "lru-cache";
 import { z } from "zod";
 
+import { BridgeChainWithDisplayInfo } from "~/components/bridge/immersive/amount-and-review-screen";
```

Comment:
IMO we should define this type here, then import type into the components. Seems bad to import stuff from the view layer into the server layer